soc/arm/nuvoton_npcx/common/pinctrl_soc.h

```diff
@@ -0,0 +1,234 @@
+/*
+ * Copyright (c) 2022 Nuvoton Technology Corporation.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef _NUVOTON_PINCTRL_SOC_H_
+#define _NUVOTON_PINCTRL_SOC_H_
```

Comment:
nit: please make this header self-contained (it makes use of DT APIs, sys/util, etc.)

Reply:
Done.